pkg/start/start.go

```diff
@@ -419,6 +443,13 @@ func (o *Options) NewControllerContext(cb *ClientBuilder) *Context {
 
 	sharedInformers := externalversions.NewSharedInformerFactory(client, resyncPeriod(o.ResyncInterval)())
 
+	// check to see if techpreview should be on or off.  If we cannot read the featuregate for any reason, it is assumed
+	// to be off.  If this value changes, the binary will shutdown and expect the pod lifecycle to restart it.
+	includeTechPreview := false
```

Comment:
oops, this API call needs to move up into `Run`, now that you're passing `includeTechPreview` into `NewControllerContext`.  [CI][1]:

```
pkg/start/start.go:448:21: no new variables on left side of :=
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/694/pull-ci-openshift-cluster-version-operator-master-images/1466836699643383808#1:build-log.txt%3A67

Reply:
fixed in c323882e.

/lgtm